codemeta.json

```diff
@@ -0,0 +1,513 @@
+{
+    "@context": "https://doi.org/10.5063/schema/codemeta-2.0",
+    "@type": "SoftwareSourceCode",
+    "license": "https://spdx.org/licenses/BSD-3-Clause",
+    "codeRepository": "git+https://github.com/gammapy/gammapy.git",
+    "contIntegration": "https://github.com/gammapy/gammapy/actions/workflows/ci.yml",
+    "dateCreated": "2014-08-25",
+    "datePublished": "2014-08-25",
+    "dateModified": "2020-11-19",
+    "downloadUrl": "https://github.com/gammapy/gammapy/archive/refs/tags/v0.18.2.tar.gz",
+    "issueTracker": "https://github.com/gammapy/gammapy/issues",
+    "name": "Gammapy",
+    "version": "0.18.2",
+    "identifier": "10.5281/zenodo.4701488",
+    "description": "Gammapy is an open-source Python package for gamma-ray astronomy built on Numpy, Scipy and Astropy. It is a prototype for the science tools of the Cherenkov Telescope Array (CTA) and is already widely used in the analysis of existing gamma-ray instruments, such as H.E.S.S., MAGIC, VERITAS and HAWC.",
+    "applicationCategory": "Astronomy",
+    "developmentStatus": "active",
+    "referencePublication": "https://ui.adsabs.harvard.edu/abs/2017ICRC...35..766D",
+    "keywords": [
+        "Astronomy",
+        "Gamma-rays",
+        "Data analysis"
+    ],
+    "programmingLanguage": [
+        "Python 3"
+    ],
+    "softwareRequirements": [
+        "Numpy, Scipy, Astropy, Regions, PyYaml, Click, Pydantic"
+    ],
+    "relatedLink": [
+        "https://gammapy.org",
+        "https://docs.gammapy.org",
+        "https://zenodo.org/record/4701500#.YJJbsC8lNpQ",
+        "https://ascl.net/gammapy"
+    ],
+    "author": [
+        {
+            "@type": "Person",
+            "givenName": "Christoph",
```

Comment:
Should we keep Christoph for V0.18.2?

Reply:
We could also use the following snippet for authors and contributors, more easy to maintain..

```
    "author": [
        {
            "@type": "Organization",
            "relatedLink": "https://gammapy.org"
        }
    ],
```

Reply:
fine with me!